handlers/profile/workspaces.py

```diff
@@ -16,11 +18,17 @@ def workspace_create(**params):
     if request.method == "POST":
         title = request.form.get("title")
         slug = request.form.get("slug")
+        user = params["user"]
+        user_id = user.get_id
 
         if title and slug:
-            Workspace.create(title=title, slug=slug)
+            workspace = Workspace.create(title=title, slug=slug)
+            workspace_id = workspace.get_id
```

Comment:
Hvala za obsežno razlago :) To sem naredil kar si napisal, samo tukaj pride do napake. Piše da tuple workspace nima vrednosti get_id. Pogledal sem pri users in zgleda enako kot sem naredu. Tudi pomoje niso potrebni parametri ker se tukaj glihkar naredi objekt workspace.

Reply:
Hey, error ti javlja zato, ker moraš tle napisat: 

```python
result, workspace, message = Workspace.create(...itd
```